src/snapshot.c

```diff
@@ -271,13 +271,13 @@ RRStatus finalizeSnapshot(RedisRaftCtx *rr, SnapshotResult *sr)
     /* Rewrite any additional log entries beyond the snapshot to a new
      * log file.
      */
-    num_log_entries = LogRewrite(rr, temp_log_filename,
-                                 rr->curr_snapshot_last_idx,
-                                 rr->curr_snapshot_last_term);
+    new_log = LogRewrite(rr, temp_log_filename,
+                         rr->curr_snapshot_last_idx,
+                         rr->curr_snapshot_last_term,
+                         &num_log_entries);
```

Comment:
@tezc, what was the reason for calling to 'LogOpen` here? only for validation? can we omit this check?

Reply:
Not sure about the original intention but we can skip it. It looks totally fine to me.

Reply:
i.e. fine with what I changed?

Reply:
@tezc